docs/user/FlowVariables.md

```diff
@@ -0,0 +1,268 @@
+| `VERILOG_FILES` | The path to the design Verilog files.                                                                                                                                |
+| `SDC_FILE`      | The path to design constraint (SDC) file.                                                                                                                            |
+| `CLOCK_PORT`    | The name of the design's clock port used in Static Timing Analysis. If your design does not have a clock port defined as an empty variable `export CLOCK_PORT= " "`. |
+| `CLOCK_PERIOD`  | The clock period for the design to meet its goal.                                                                                                                    |
+
+
+### Optional Variables
+
+
+These are optional variables that may be over-ridden/appended with
+default value from the platform `config.mk` by defining in the design
+configuration file.
+
+
+#### Synthesis
+
+
+| Variable                 | Description                                                                                        |
+|--------------------------|----------------------------------------------------------------------------------------------------|
+| `ADDITIONAL_LEFS`        | Hardened macro LEF view files listed here.                                                         |
+| `ADDITIONAL_LIBS`        | Hardened macro library files listed here.                                                          |
+| `ADDITIONAL_GDS`         | Hardened macro GDS files listed here.                                                              |
+| `VERILOG_INCLUDE_DIRS`   | Specifies the include directories for the Verilog input files.                                     |
+| `VERILOG_FILES_BLACKBOX` | Hardened macro Verilog file list to be added here.                                                 |
+| `CORNER`                 | PVT corner library selection fast/typical/worst.                                                   |
+| `DFF_LIB_FILE`           | Define this variable if the design has asynchronous clocks.                                        |
+| `DESIGN_NICKNAME`        | If the design has a hierarchy, to select a specific module as top design this variable is defined. |
+| `ABC_CLOCK_PERIOD_IN_PS` | Clock period to be used by STA during synthesis. Default value read from SDC.                      |
+| `ABC_AREA`               | Strategies for Yosys ABC synthesis: Area/Speed. Default ABC_SPEED.                                 |
+| `PWR_NETS_VOLTAGES`      | Used for IR Drop calculation.                                                                      |
+| `GND_NETS_VOLTAGES`      | Used for IR Drop calculation.                                                                      |
+
+
+#### Floorplanning

```

Comment:
Why do you have both Floorplan and Floorplanning?

Reply:
While planning this document our idea is give list of platform config.mk defined variable as one list.
Next design specific config.mk variables list.
List of optional variables that user can overwrite.
Please add your review to keep this optional variables list or not?
@maliberty